backend/model/message/src/main/java/co/airy/model/message/dto/MessageResponsePayload.java

```diff
@@ -0,0 +1,44 @@
+package co.airy.model.message.dto;
+
+import co.airy.avro.communication.Message;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import static co.airy.date.format.DateFormat.isoFromMillis;
+import static co.airy.model.message.MessageRepository.resolveContent;
+
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class MessageResponsePayload {
+    private String id;
```

Comment:
cc @AitorAlgorta  @lucapette there was a conversation about having the `source` field on every message that comes in so that they can be rendered even without the conversation (e.g. websocket). Is that still relevant?

Reply:
Good point!